torch/fx/experimental/validator.py

```diff
@@ -277,12 +277,37 @@ def wrapper(*args):
 
             return wrapper
 
+        # Like `lift`, but for bitwise ops. We need to convert to/from BitVec
+        # in order to use z3 bitwise ops. We assume that integers are 64 bit.
```

Comment:
Safe to assume?

Reply:
We don't currently make this assumption I think, but it might also be harmless here. cc @ysiraichi

Reply:
Probably safe to assume. But, not sure this will play well with the other operations. Could you add a test where there is a, for example, bitwise operation followed by, say, `mod`?